include/crocoddyl/core/solvers/ipopt/ipopt.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef CROCODDYL_CORE_SOLVERS_IPOPT_IPOPT_HPP_
+#define CROCODDYL_CORE_SOLVERS_IPOPT_IPOPT_HPP_
+
+#include "coin-or/IpIpoptApplication.hpp"
```

Comment:
```suggestion
#include <coin-or/IpIpoptApplication.hpp>
```

This is part of our code style.